libcontainer/nsenter/nsexec.c

```diff
@@ -866,39 +866,43 @@ void send_fd(int sockfd, int fd)
 		bail("failed to send fd %d via unix socket %d", fd, sockfd);
 }
 
-void receive_mountsources(int sockfd)
+void receive_from_env(int sockfd, char *env_variable)
```

Comment:
Is this name is too generic? We don't just receive anything, this function is VERY specific on what it can receive